src/partials/messenger.conversation.html

```diff
@@ -22,7 +24,28 @@
                 <span ng-bind-html="ctrl.receiver.members | idsToNames | escapeHtml | emojify"></span>
             </div>
         </div>
+
+        <!-- Menu -->
+        <div class="header-buttons">
+            <md-button
```

Comment:
Mhh... would it make sense to move this into a directive? The two buttons seem awkward to me. Perhaps this could even be reused for further toggle buttons.

Reply:
:+1: